proxy/proxy/spacewalk-proxy.changes

```diff
@@ -1,3 +1,5 @@
+- Added passing a component type property to the LOG object 
+  when initializing the logger 
```

Comment:
Something like this is more user-friendly. Remember changelogs are not for developers.
```suggestion
- Prefix log messages with the component name to ease analysis
```